### PR TITLE
add sha256 support if available on bosh.io

### PR DESCRIPTION
**Pull request by ramonskie**
adding sha256 support
depends on bosh.io PR https://github.com/bosh-io/web/pull/52

### CODE REVIEW

**Comment by taylorsilva** [comment]
@ramonskie I'll come back to this PR every so often since it's assigned to me. Feel free to ping me still whenever your bosh.io PR gets merged.

**Comment by ramonskie** [comment]
@taylorsilva the pr has been merged and bosh.io now serves sha256

File: cmd/in/main.go [hunk 1]
```diff
@@ -65,7 +65,7 @@ func main() {
 		log.Fatalf("failed to find stemcell matching version: '%s'\n", inRequest.Version.Version)
 	}
 
-	dataLocations := []string{"version", "sha1", "url"}
+	dataLocations := []string{"version", "sha1", "sha256", "url"}
```
Comment (taylorsilva) [inline cmd/in/main.go:68]: Do you want to update the README to mention that there's a `sha256` file for `in`/get steps?

**Review by taylorsilva — CHANGES_REQUESTED** [review]

File: cmd/in/main.go (hunk 1 above)
Comment (ramonskie) [reply cmd/in/main.go:68]: i already mentioned it in the readme
https://github.com/concourse/bosh-io-stemcell-resource/pull/27/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R42

Comment (taylorsilva) [reply cmd/in/main.go:68]: ah derp, my bad! I had hit the "view" checkbox and by the time I got to the last file here I completely forgot about that file! We're all good then!

**Review by taylorsilva — APPROVED** [review]

**Pull request merged**